src/supabase/queries.ts

```diff
@@ -0,0 +1,45 @@
+import { supabase } from './client';
+
+// Function to add a single tree
+export async function addTree(species: string) {
+  const { error } = await supabase.rpc('add_tree', { species });
+
+  if (error) {
+    throw new Error(`Error adding tree: ${error.message}`);
+  }
+}
+
+// Function to add multiple trees
+export async function addMultipleTrees(
```

Comment:
When I run `addMultipleTrees([{ species: 'Big Tree', quantity: 4 }])`(lmk if i'm calling the function wrong) i'm seeing `Error adding multiple trees: Could not find the function public.add_multiple_trees(trees) in the schema cache`. I think it could be bc the db function takes in species and quantity separately and this function only passes in a `trees` argument? I'm not too familiar with db functions so idk

Reply:
resolved via refactoring